chapter02/exercises/ex01/main.cpp

```diff
@@ -10,7 +10,11 @@
 int main()
 {
     std::string address = "Gantiadi, 24";
-    std::string name = "Nadia Sitkareva";
+    std::string first_name = "Nadia";
```

Comment:
Please consider to use 'user' prefix before your variables, as foolows:
user_first_name
user_last_name
user_address

Reply:
done